app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
```

Comment:
Nghiên cứu i18n trong rails để đứa The và Ruby on Rails Tutorial vào file en.yml  em nhé